### DISCUSSION TITLE
FormSheet Background Opacity

### DISCUSSION THREAD

**Discussion by JacobJaffe** [react-navigation/react-navigation#12325] [Q&A]
Hello, when using `presentation: "formSheet"` in v7, is there a way to control the background opacity?

(on iOS)

Previously, when using modals, I would do this with an Animated component component covering the whole screen. However, it looks like the `formSheet` has a *bit* of background contrasting baked in already.

I'd like to bump this up, especially when using detents, so there's more visible contrast between a half-open formSheet and the underlying screen. 

Example:
```tsx
options: {
    presentation: "formSheet",
    sheetAllowedDetents: [0.5, 1.0],
    sheetInitialDetentIndex: 0, 
    sheetBackgroundOpacity: 0.5 // <--- is there a way to achieve this kind of functionality?
}
```

**Comment by satya164** [comment]
cc @kkafar

**Comment by JacobJaffe** [comment]
Looks like this might need to be exposed in the react-native-screens layer so also posted there:
https://github.com/software-mansion/react-native-screens/discussions/2555

(Sorry for double discussion!)
